terraform/appflask.py

```diff
@@ -703,6 +707,12 @@ def cluster_view():
 
         return render_template('cluster.html', api_ip=api_ip, k8s_ver=k8s_versions())
 
+def calculate_k8s_as(fabric_as):
+    '''Ensure the K8s AS number falls within 1 and 65534'''
+    fabric_as = int(fabric_as)
+    if fabric_as >= 65534:
```

Comment:
What if it well and truly above 65534? For example : 70000 would become 69999?
When it should really be 65533. 
This would be great function to unittest.